projects/packages/videopress/src/client/block-editor/blocks/video/style.native.scss

```diff
@@ -9,5 +9,13 @@
 	width: 100%;
 	// TODO: Temporarily setting height to 250px so placeholder is visible.
```

Comment:
I believe this `TODO` could be removed now?

Reply:
Good point. Seems we can remove these comments.

Reply:
fixed in 2c936630a3cbaacf64977922c58ce383b376d49c